app/models/bishop.rb

```diff
@@ -10,4 +10,12 @@ def valid_move?(new_rank, new_file)
     
     !is_move_obstructed? new_rank, new_file
   end
+
+  def image
+    if color == 'white'
+      return '&#9815;'
+    else
+      return '&#9821;'
```

Comment:
I like the use of Unicodes here. Hint - they black is +6 from white, so something like this could reduce repetition:

```
mod = color == 'white' ? 0 : 6
return "&##{9815 + mod}"
```